paraatm/HPMM/hpmm.py

```diff
@@ -0,0 +1,101 @@
+#! /usr/bin/python3 python3
+# -*- coding: utf-8 -*-
+
+"""
+@Author: Jiawei Chen
+@Date: 2020-05-20
+This Python script is used for HPMM (Human Performance Monitoring Module)
+The script outputs predicted loss of separation as an indicator of air traffic controller's performance
+The script can also check compliance for each command
+@Last Modified by: Jiawei Chen
+@Last Modified date: 2020-05-26
+"""
+from typing import Any, Union
+
+from jpype import *
+from array import *
```

Comment:
Please remove use of `import *`, which makes it difficult to know which packages the functions that you use in your code come from, or which of your imports are actually used in the code.